### DISCUSSION TITLE
general chat

### DISCUSSION THREAD

**Discussion by flauschzelle** [timhutton/twitter-archive-parser#129] [General]
Hi :) 
Just testing if this could work as a chat. But I think it's more of a forum than a chat?

**Comment by lenaschimmel** [comment]
Well, I think that depends on if it updates in realtime on your screen if I answer?

**Comment by timhutton** [comment]
I've started putting some thoughts about the scope and overall approach of the project onto the wiki.

I'm interested to hear what coding style / design approach changes you would recommend, that meet those guidelines. The need to jam everything into one file is getting quite annoying but I can't currently see a way around that.

**Comment by LouBdt** [comment]
Just committed a little script that reorganizes the .md files of tweets out of your parser, and finds the threads
The repository is [here](https://github.com/LouBdt/twitter-post-parser)
Would love to have some feedback

**Comment by mlevin77** [comment]
I'm sorry if this is a stupid question, but can this code produce a single long PDF or something that can be printed into a single long page of all the tweets?  I've got my archive, I've run your code, but it looks like it created a ton of individual .html files for specific time periods - am I missing a single html where one can scroll through the complete series of tweets?  thank you in advance!

**Comment by wapatoecology** [comment]
Thanks so much for this. I just ran it on my archive and I imported it into an obsidian archive. totally works.

However, one question: it doesn't seem to preserve the links between tweets that make up threads.
Did it preserve "in_reply_to_status_id" ?

**Reply by wapatoecology** [reply]
ok i see it's linking it back to the original tweet on twitter. any way to set this up so that it links to the tweet in md? i have a ton of threads I'm trying to archive
